### ISSUE TITLE
License issue with free Wolfram Engine and kernel limits

### ISSUE THREAD

**Issue by liveod3** [njpipeorgan/wolfram-language-notebook#22]
**Activated license expired**

I installed the free Wolfram Engine in Windows Subsystem of Linux, Ubuntu 20.04.3 LTS and tried using it with this extension in VSCode. However, each time after using it for a while, wolframscript prompts that the license is no long valid. 

The Problem might be the kernel limit.  

When I'm using wolfram-language-notebook, command

> pgrep wolfram 

returns 2 pid.

When I'm using wolframscript in command-line, command

> pgrep wolfram

returns 1 pid.

**Environment**
 - Operating system: Windows11 Pro 21H2
 - Mathematica/Engine: 13.0.1
 - VS Code: 1.65.2
 - Version: 0.0.9

**Comment by njpipeorgan** [comment]
Thanks for the feedback. It is normal to find two wolfram processes running--one process is responsible for communication and the other one for computation. The latter process is launched as a subkernel, so it should not trigger the kernel limit problem in principle. 

Where did you get the "license no longer valid" message?

**Comment by liveod3** [comment]
At first the extension works fine, but then it indicates me that it failed to connect to the kernel. So I checked the output of the extension.  The kernel returns a piece of error message says that my wolfram account is not valid for activating wolfram engine because I might have activated the engine twice whereas I haven't. I also tried wolframscript command in powershell and got the same result. 

I guess the problem might relate to the environment instead of kernels and the extension.

Thanks for the explanation.
